insights/parsers/ls_var_lib_rsyslog.py

```diff
@@ -0,0 +1,28 @@
+"""
+LsVarLibRsyslog - command ``ls -lZ  /var/lib/rsyslog``
+======================================================
+"""
+
+
+from insights.specs import Specs
+from insights import CommandParser, parser, FileListing
```

Comment:
Could you please change these to full path imports like below?

```
from insights.core import CommandParser, FileListing
from insights.core.plugins import parser
from insights.specs import Specs
```

Reply:
> from insights.core import CommandParser, FileListing
> from insights.core.plugins import parser
> from insights.specs import Specs

@ryan-blakley updated, please help to check, thanks.

Reply:
Thank you, looks good.